components/playlist/browser/playlist_service.cc

```diff
@@ -622,7 +622,10 @@ void PlaylistService::ReorderPlaylist(const std::string& playlist_id,
                                       ReorderPlaylistCallback callback) {
   {
     ScopedListPrefUpdate playlist_order_update(prefs_, kPlaylistOrderPref);
-    auto it = base::ranges::find(*playlist_order_update, playlist_id);
+    auto it = std::ranges::find_if(
+        *playlist_order_update, [&playlist_id](const base::Value& value) {
+          return value.is_string() && value.GetString() == playlist_id;
+        });
     if (it == playlist_order_update->end()) {
```

Comment:
Would `auto it = std::ranges::find(*playlist_order_update, base::Value(playlist_id));` work?

Reply:
I thought of it, but ultimately didn't want to because it will lead to a copy/instantiation.